python/ray/data/preprocessors/concatenator.py

```diff
@@ -42,61 +45,22 @@ class Concatenator(Preprocessor):
         By default, the created column is called `"concat_out"`, but you can specify
         a different name.
 
-        >>> concatenator = Concatenator(output_column_name="tensor")
+        >>> concatenator = Concatenator(columns=["X0", "X1"], output_column_name="tensor")
         >>> concatenator.fit_transform(ds).to_pandas()  # doctest: +SKIP
                tensor
         0  [0.0, 0.5]
         1  [3.0, 0.2]
         2  [1.0, 0.9]
 
-        Sometimes, you might not want to concatenate all of of the columns in your
-        dataset. In this case, you can exclude columns with the ``exclude`` parameter.
-
-        >>> df = pd.DataFrame({"X0": [0, 3, 1], "X1": [0.5, 0.2, 0.9], "Y": ["blue", "orange", "blue"]})
-        >>> ds = ray.data.from_pandas(df)  # doctest: +SKIP
-        >>> concatenator = Concatenator(exclude=["Y"])
-        >>> concatenator.fit_transform(ds).to_pandas()  # doctest: +SKIP
-                Y  concat_out
-        0    blue  [0.0, 0.5]
-        1  orange  [3.0, 0.2]
-        2    blue  [1.0, 0.9]
-
-        Alternatively, you can specify which columns to concatenate with the
-        ``include`` parameter.
-
-        >>> concatenator = Concatenator(include=["X0", "X1"])
-        >>> concatenator.fit_transform(ds).to_pandas()  # doctest: +SKIP
-                Y  concat_out
-        0    blue  [0.0, 0.5]
-        1  orange  [3.0, 0.2]
-        2    blue  [1.0, 0.9]
-
-        Note that if a column is in both ``include`` and ``exclude``, the column is
-        excluded.
-
-        >>> concatenator = Concatenator(include=["X0", "X1", "Y"], exclude=["Y"])
-        >>> concatenator.fit_transform(ds).to_pandas()  # doctest: +SKIP
-                Y  concat_out
-        0    blue  [0.0, 0.5]
-        1  orange  [3.0, 0.2]
-        2    blue  [1.0, 0.9]
-
-        By default, the concatenated tensor is a ``dtype`` common to the input columns.
-        However, you can also explicitly set the ``dtype`` with the ``dtype``
-        parameter.
-
-        >>> concatenator = Concatenator(include=["X0", "X1"], dtype=np.float32)
+        >>> concatenator = Concatenator(columns=["X0", "X1"], dtype=np.float32)
         >>> concatenator.fit_transform(ds)  # doctest: +SKIP
         Dataset(num_rows=3, schema={Y: object, concat_out: TensorDtype(shape=(2,), dtype=float32)})
 
     Args:
         output_column_name: The desired name for the new column.
             Defaults to ``"concat_out"``.
-        include: A list of columns to concatenate. If ``None``, all columns are
-            concatenated.
-        exclude: A list of column to exclude from concatenation.
-            If a column is in both ``include`` and ``exclude``, the column is excluded
-            from concatenation.
+        columns: A list of columns to concatenate. The order will define the order
+            of the concatenated columns.
```

Comment:
```suggestion
        columns: A list of columns to concatenate. The order of the columns will be 
            retained during concatenation.
```

Reply:
Also - should we just keep it as `include`?

Reply:
ok with keeping it, but just want to point out that it would be still a breaking change, and that other preporcessors use `columns` to define the columns that the preprocessor would be apply to

Reply:
it will be a required argument right now, if non is provided it will fail

Reply:
discussed offline - this makes sense and columns is more consistent with other preprocessors as well.